packs/KotI.json

```diff
@@ -0,0 +1,197 @@
+{
+    "cgdbId": 45,
+    "code": "KotI",
+    "name": "King of the Isles",
+    "releaseDate": null,
+    "cards": [
+        {
+            "code": "12001",
+            "type": "character",
+            "name": "Balon Greyjoy",
+            "unique": true,
+            "faction": "greyjoy",
+            "loyal": true,
+            "cost": 7,
+            "icons": {
+                "military": true,
+                "intrigue": false,
+                "power": true
+            },
+            "strength": 5,
+            "traits": [
+                "Ironborn",
+                "Lord"
+            ],
+            "text": "Pillage. Renown.\n<b>Action:</b> Kneel your faction card to choose a card in an opponent's discard pile and put it into play under your control. At the end of the phase, shuffle that card into its owner's deck.",
+            "deckLimit": 3
+        },
+        {
+            "code": "12003",
+            "type": "character",
+            "name": "Asha Greyjoy",
+            "unique": true,
+            "faction": "greyjoy",
+            "loyal": false,
+            "cost": 6,
+            "icons": {
+                "military": true,
+                "intrigue": false,
+                "power": true
+            },
+            "strength": 5,
+            "traits": [
+                "Captain",
+                "Ironborn",
+                "Lady"
+            ],
+            "text": "Pillage. Stealth.\nEach other unique <i>Ironborn</i> character you control gains stealth.",
+            "deckLimit": 3
+        },
+        {
+            "code": "12012",
+            "type": "character",
+            "name": "Priest of Old Wyk",
+            "unique": false,
+            "faction": "greyjoy",
+            "loyal": false,
+            "cost": 3,
+            "icons": {
+                "military": false,
+                "intrigue": false,
+                "power": true
+            },
+            "strength": 4,
+            "traits": [
+                "Drowned God"
+            ],
+            "text": "While there is a <i>Drowned God</i> character in your dead pile, Priest of Old Wyk contributes its STR to your total for dominance even while kneeling.",
+            "deckLimit": 3
+        },
+        {
+            "code": "12014",
+            "type": "character",
+            "name": "Eager Deckhand",
```

Comment:
Do we have confirmation on the ID number for this card? It's obscured in the preview image and since it's a full expansion, it isn't easy to guess like it would be in packs.

It's probably safer to leave it out of the data for now unless we're sure, rather than put it in with the wrong number, have decks built on ThronesDB and Throneteki using it, and then having those decks suddenly swap cards when we get the correct number.

Reply:
Nope, was simply an educated guess. I've removed the entry.